doc/user/nhrpd.rst

```diff
@@ -106,11 +156,12 @@ with:
 To start sending these traffic notices out from hubs, use the nhrp
 per-interface directive:
 
-.. code-block:: frr
-
-   interface gre1
-    ip nhrp redirect
+.. index::  ip nhrp redirect
+.. clicmd:: ip nhrp redirect
 
+This enable redierct replies on the NHS similiar to ICMP redirects excpet this is
```

Comment:
typo in `redierct`

Reply:
typo in `similiar`

Reply:
typo in `excpet` :)

Reply:
forgot to run the spell checker in my emacs session! thanks @ton31337